.github/workflows/test.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           tar xfv pwru-linux-${{ matrix.arch }}.tar.gz
 
       - name: Provision LVH VMs
-        uses: cilium/little-vm-helper@9c1f3a549af06e213863d034c13ba1c5d1e3c667 # v0.0.26
+        uses: cilium/little-vm-helper@01debd6cb7e5514cfdb4a33e776bdc647bc5306e # v0.0.27
         with:
           lvh-version: 'v0.0.24'
```

Comment:
@brb Any idea why this isn't getting updated? I see from the CI logs that we pull lvh images with the v0.0.24 version instead of v0.0.27

Reply:
Oh, that's much nicer than what I was trying :triangle-of-success: